napari/components/layerlist.py

```diff
@@ -64,6 +62,16 @@ def __init__(self, data=()):
             basetype=Layer,
             lookup={str: lambda e: e.name},
         )
+
+        # TODO: figure out how to move this context creation bit.
```

Comment:
Just noting +1 to this - thanks for leaving ToDo

Reply:
yep, it's also in the list here: https://github.com/napari/napari/issues/4860